src/vishakha/TESTP1_FirstNonRepeatingChar.java

```diff
@@ -0,0 +1,33 @@
+package vishakha;
+
+public class TESTP1_FirstNonRepeatingChar {
+	int stringOccur(String input, char ch) {
+		int count = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (input.charAt(index) == ch)
+				count++;
+		}
+		return count;
+	}
+
+	void firstSingleOccur(String input) {
+		char c = ' ';
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			int count = stringOccur(input, ch);
+			if (count == 1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop